src/main/kotlin/com/bardsoftware/papeeria/backend/cosmas/CosmasServer.kt

```diff
@@ -50,12 +51,17 @@ class CosmasServer(port: Int) {
 fun main(args: Array<String>) {
     val arg = CosmasServerArgs(ArgParser(args))
     println("Try to bind in port ${arg.port}")
-    val server = CosmasServer(arg.port)
+    val server =
+            if (arg.bucket != "")
+                CosmasServer(arg.port, CosmasGoogleCloudService(arg.bucket))
+            else
+                CosmasServer(arg.port, CosmasInMemoryService())

```

Comment:
Please always use curly braces in the control flow statements. The only exception is when you write the whole statement on a single line: `if (condition) then doThis() else doThat()`

The reason is that code without curly braces is way more error prone. Should you insert a single logging line above this line, and your code turns into a pumpkin.

Reply:
done